src/engine/strat_engine/filesystem.rs

```diff
@@ -152,7 +152,7 @@ pub fn mount_fs<P1, P2>(devnode: &P1, mountpoint: &P2) -> EngineResult<()>
 {
     Ok(try!(mount(Some(devnode),
                   mountpoint,
-                  None as Option<&Path>,
+                  Some("xfs"),
```

Comment:
If this does actually fix the intermittent issue then I think the commitmsg can be a little more assertive in saying so. Does it?